Web Quiz Engine (Kotlin)/task/src/engine/entity/QuizCompletion.kt

```diff
@@ -0,0 +1,28 @@
+package engine.entity
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import engine.QuizCompletionIdentifier
+import jakarta.persistence.*
+import java.time.ZonedDateTime
+
+@Entity
+@Table(name = "completion")
+data class QuizCompletion(
+    @Id
+    @GeneratedValue
+    val id: QuizCompletionIdentifier? = null,
```

Comment:
I'm guessing the CodeLab came with this? Not sure where `QuizCompletionIdentifier` is coming from.
Anyway this is a good way to implement composite keys 👍 .

Reply:
Well, actually this is a `typealias` on `Int` `WebQuizCompletionRepository`